cylc/flow/scripts/cylc.py

```diff
@@ -214,6 +216,8 @@ def get_version(long=False):
     'checkpoint':
         'DB checkpoints have been removed, use a reflow to '
         '"rewind" a workflow.',
+    'conditions':
+        'cylc conditions has been replaced by cylc help license',
```

Comment:
I'd be surprised if anyone ever typed `cylc conditions` :grin:

Reply:
Ditto.

Was a result of going through the old CLI when writing the migration guide stuff.